app/controllers/concerns/presentable.rb

```diff
@@ -3,7 +3,13 @@
 module Presentable
   extend ActiveSupport::Concern
 
-  def room_presentation(room)
-    return url_for(room.presentation) if room.presentation.attached?
+  def presentation_name(room)
+    return room.presentation.filename if room.presentation.attached?
+  end
+
+  def presentation_representation(room)
+    return unless room.presentation.attached? && room.presentation.representable?
+
+    rails_representation_url(room.presentation.representation(resize: ['500x500']).processed)
```

Comment:
Why are we resizing to 500x500 here? Shouldn't we just resize directly to the size we want?

Reply:
It doesn't actually change the size of the picture right away, it just changes the resolution (I know it's weird).

I will change it to match the size that we are changing it to though in the `presentation.scss`